interfaces/sdcore_config/v0/interface_tests/test_requirer.py

```diff
@@ -5,33 +5,17 @@
 from scenario import Relation, State
 
 
-def test_state_active_on_joined():
+def test_no_data_on_joined():
```

Comment:
there should be a 'happy-path' test in here too, one where you `t.assert_schema_valid()`.